ddtrace/span.py

```diff
@@ -206,6 +211,9 @@ def finish(self, finish_time=None):
             if self.tracer and trace and sampled:
                 self.tracer.write(trace)
 
+        for cb in self._on_finish_callbacks:
+            cb(self)
```

Comment:
No error handling there?

Reply:
I'm not sure what the error handling would accomplish. If the callback raised then I think that'd be the responsibility of the own of a `Span` to handle.